mentat/session.py

```diff
@@ -245,6 +246,7 @@ async def run_main():
                 # Helps us handle errors in tests
                 if is_test_environment():
                     print(error)
+                self.error = error
```

Comment:
I'd like the python client to print the session's error when it fails for easier debugging. I'm not sure this is actually the best way to accomplish that. One alternative would be to have the the above check evaluate to true for benchmark environments which would give us most of the benefit.